tests/sqlparser_postgres.rs

```diff
@@ -834,6 +834,11 @@ fn parse_comments() {
     }
 }
 
+#[test]
+fn parse_quoted_identifier() {
+    pg_and_generic().verified_stmt(r#"SELECT "quoted "" ident""#);
```

Comment:
perhaps we can add a test for dialects (e.g. mysql) that this isn't parsed as an identifier?

Reply:
I added a test for ` quoted strings in mysql